infrastructure/staging_stack.py

```diff
@@ -1,13 +1,13 @@
 from typing import Any
 
-from aws_cdk import aws_s3, aws_ssm
+from aws_cdk import aws_s3
 from aws_cdk.core import Construct, NestedStack, RemovalPolicy, Tags
 
-from backend.parameter_store import ParameterName
+from backend.resources import ResourceName
 
 
 class StagingStack(NestedStack):
-    def __init__(self, scope: Construct, stack_id: str, *, deploy_env: str, **kwargs: Any) -> None:
+    def __init__(self, scope: Construct, stack_id: str, **kwargs: Any) -> None:
```

Comment:
I think we can also remove `**kwargs`, since we never actually pass any extra parameters.